nva-datamodel-java/src/main/java/no/unit/nva/model/associatedartifacts/file/File.java

```diff
@@ -232,15 +263,18 @@ private String getLicenseValue(Map<?, ?> license) {
     }
 
     /**
-     * Validate license.
+     * Validates and formats a provided URI license.
+     *
+     * @param license the URI license to be validated
+     * @return the formatted license if the provided license is valid and not a "RightsReserved" license, or the
+     *     original license if it's valid and a "RightsReserved" license
+     * @throws IllegalArgumentException if the license is null or not valid
      */
     private URI validateUriLicense(URI license) {
-        if (nonNull(license) && isValidUriLicense(license) && !license.equals(LICENSE_MAP.get("RightsReserved"))) {
-            return formatValidUriLicense(license);
-        } else {
-            logger.info("The specified license can not be converted into valid URI license: {}", license);
+        if (license == null || !isValidUriLicense(license) || license.equals(LICENSE_MAP.get("RightsReserved"))) {
```

Comment:
use `nonNull(license)`

Reply:
I want to checkk for `null`. Changed to `isNull(license)`